addons/product/models/product_pricelist.py

```diff
@@ -221,21 +204,28 @@ def _compute_price_rule(self, products, qty, uom=None, date=False):
         return results
 
     # Split methods to ease (community) overrides
-    def _get_applicable_rules(self, *args, **kwargs):
+    def _get_applicable_rules(self, products, date, **kwargs):
         self.ensure_one()
         # Do not filter out archived pricelist items, since it means current pricelist is also archived
         # We do not want the computation of prices for archived pricelist to always fallback on the Sales price
         # because no rule was found (thanks to the automatic orm filtering on active field)
         return self.env['product.pricelist.item'].with_context(active_test=False).search(
-            self._get_applicable_rules_domain(*args, **kwargs)
+            self._get_applicable_rules_domain(products=products, date=date, **kwargs)
         )
 
-    def _get_applicable_rules_domain(self, date, product_tmpl_ids, product_ids, category_ids):
+    def _get_applicable_rules_domain(self, products, date):
```

Comment:
Seems the **kwargs are missing here for clean community overrides, could be fixed :eyes: